src/transformers/models/marian/tokenization_marian.py

```diff
@@ -310,7 +310,7 @@ def __setstate__(self, d: Dict) -> None:
         self.current_spm = self.spm_source
         self._setup_normalizer()
 
-    def num_special_tokens_to_add(self, **unused):
+    def num_special_tokens_to_add(self, *args, **kwargs):
```

Comment:
This change is required to accommodate the use of positional arguments like `tokenizer.num_special_tokens_to_add(is_pair)` in `_generate_dummy_inputs_for_sequence_classification_and_question_answering()`.

I'm not sure why we had `**unused` in the first place, but the change also seems more conventional IMO.